lib/internal/modules/esm/worker.js

```diff
@@ -74,7 +74,9 @@ function wrapMessage(status, body) {
 }
 
 async function customizedModuleWorker(lock, syncCommPort, errorHandler) {
-  let hooks, preloadScripts, initializationError;
+  let hooks;
+  let preloadScripts;
+  let initializationError;
```

Comment:
Unrelated change I think, what's the point of it?

Reply:
I assume this was part of a copy-paste from my PR, in which `preloadScripts` was removed:

from:
```js
  let hooks, preloadScripts, initializationError;
```

to:

```js
  let hooks;
  let initializationError;
```

I was touching that line already (and the former was more difficult to grok), so breaking them into multiple lines seemed fair game.

Reply:
I disagree, either we make a lint rule to enforce using several lines, either we leave it as is. At the end of the day, this is just a cosmetic change, which we typically reject to not pollute git blame and avoid an endless debate with interposed PR on what style is "better". Let's not have it in this PR.

Reply:
> Let’s please not argue about style. The way I have it here is better for avoiding conflicts, which is the goal of this PR.

If you want us to not argue, you should remove the changes in `lib/` and make this PR ``test: isolate `globalPreload` tests``. Sorry not sorry, the way you have here is worse for git blame, which is a personal goal of mine – if that means we might have to fix a one-line conflict in the hypothetical case where we would need to touch those lines again, so be it.

Reply:
What goal are you trying to achieve regarding `git blame`?

The only changes in `lib/` are this one and adding a missing property from JSDoc, which is completely unrelated to `globalPreload` removal and should just land.

Reply:
> The only changes in `lib/` are this one and adding a missing property from JSDoc, which is completely unrelated to `globalPreload` removal

Yeah, I agree, that's why it doesn't belong in this PR. It can land as its own PR.

> What goal are you trying to achieve regarding `git blame`?

Trying to keep that feature useful. If we start landing commits that changes unrelated lines, it increases the noise and decrease the general usefulness of git blame. I would suggest you to try to open a PR specific to those lines, but that's unlikely to ever land because we don't accept PRs that only do cosmetic changes.
Anyway I'm -1 on this change. Let me know if you want me to take it over.

Reply:
The line that’s currently `let hooks, preloadScripts, initializationError;` will change either in this PR or in #49144, so whatever `blame` history it currently has is about to be lost, either by this PR or that one. So there’s nothing to preserve there. Splitting it into three lines reduces the potential for backport conflicts. Keeping it as one line seems to me to have no advantages.